hack/util.sh

```diff
@@ -458,10 +458,15 @@ function install_registry() {
 }
 readonly -f install_registry
 
-function wait_for_registry() {
+function is_registry_ready() {
 	local generation="$(oc get dc/docker-registry -o 'jsonpath={.metadata.generation}')"
 	local onereplicajs='{.status.observedGeneration},{.status.replicas},{.status.updatedReplicas},{.status.availableReplicas}'
-	wait_for_command "oc get dc/docker-registry -o 'jsonpath=${onereplicajs}' --config='${ADMIN_KUBECONFIG}' | grep '^${generation},1,1,1$'"  "$((5*TIME_MIN))"
+	oc get dc/docker-registry -o "jsonpath=${onereplicajs}" --config="${ADMIN_KUBECONFIG}" | fgrep -qx "${generation},1,1,1"
```

Comment:
I also feel like there's no good reason to conflate these four tests with each other -- we want to test:
 - `observedGeneration == dc.generation`
 - `replicas == 1`
 - `updatedReplicas == 1`
 - `availableReplicas == 1`

We aren't hurting for API calls, right? I would pefer to have four tests instead of one that sticks commas in there. I won't block this PR on it, though.